spec/requests/users_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+describe 'Password Reset Requests', type: :request do
+  let(:endpoint) { "/api/users/#{user.id}/reset_password/" }
+  let(:user) { create(:user) }
+  let(:other_user) { create(:user) }
+  let(:prr) { create(:unused_password_reset_request, user: user) }
+  let(:new_password) { 'NEW PASSWORD$$$?!?!?' }
+  let(:reset_params) do
+    {
+      password_reset_request_id: prr.id,
+      password: new_password
+    }
+  end
+
+  describe 'POST /api/users/:id/reset_password/' do
+    it 'successfully resets a password' do
+      post endpoint, params: reset_params.to_json, headers: unauthenticated_header
+
+      expect(response).to have_http_status(204)
+      expect(BCrypt::Password.new(user.reload.password_digest)).to eq(new_password)
+      expect(prr.reload.used).to be(true)
```

Comment:
`be_truthy` matches `true`, `1`, `'some string'`, etc. I'm looking specifically for `true`